example/lib/pages/posts_list/posts_list_page_scenes.dart

```diff
@@ -54,25 +62,29 @@ class WithPostsScene extends BasePostsListScene {
   }
 }
 
+/// A Scene showing the [PostsListPage] in a loading state.
 class LoadingScene extends BasePostsListScene {
   @override
   String get title => 'Loading';
 
   @override
   Future<void> setUp() async {
     await super.setUp();
-    final completer = Completer<List<Post>>();
+    final Completer<List<Post>> completer = Completer<List<Post>>();
```

Comment:
It's surprising to me that this is necessary...

Reply:
Yup, :( https://dart-lang.github.io/linter/lints/always_specify_types.html